frontend/src/component/common/PercentageCircle/DisabledPercentageCircle.tsx

```diff
@@ -0,0 +1,46 @@
+import { useTheme } from '@mui/material';
+import { CSSProperties } from 'react';
+
+interface IPercentageCircleProps {
+    percentage: number;
+    size?: `${number}rem`;
+}
+
+const PercentageCircle = ({
+    percentage,
+    size = '4rem',
+}: IPercentageCircleProps) => {
+    const theme = useTheme();
+
+    const style: CSSProperties = {
+        display: 'block',
+        borderRadius: '100%',
+        transform: 'rotate(-90deg)',
+        height: size,
+        width: size,
+        background: theme.palette.background.elevation2,
+    };
+
+    // The percentage circle used to be drawn by CSS with a conic-gradient,
+    // but the result was either jagged or blurry. SVG seems to look better.
+    // See https://stackoverflow.com/a/70659532.
+    const r = 100 / (2 * Math.PI);
+    const d = 2 * r;
```

Comment:
Can we use full names here please? Radius and diameter are much more readable than r & d.